src/components/Capture/ProfileData.tsx

```diff
@@ -92,6 +94,7 @@ const ProfileData = ({
             type={type as FieldComponentProps['type']}
             value={value as FieldComponentProps['value']}
             selectedCountry={formData.country}
+            disabled={(disabledFields || []).includes(type)}
```

Comment:
the dropdown is disabled but the value is not set

Reply:
Fixed 👍